images/minio/config/template.apko.yaml

```diff
@@ -0,0 +1,24 @@
+contents:
+  packages:
+    - minio
+    - minio-oci-entrypoint
+    - busybox
```

Comment:
```suggestion
```

you can remove this since it's a package dependency now

Reply:
I think I can remove both, as the entry script was just creating a user that we're already doing